### DISCUSSION TITLE
beta50 升到 60 提示 [vite] Internal server error: Missing "./layouts/Layout.vue" export in "@vuepress/theme-default" package

### DISCUSSION THREAD

**Discussion by cgx452306705** [vuepress/core#1267] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/12523996/220263647-738845ce-43dc-48a3-82c1-c0d443758f8a.png)

**Comment by Mister-Hope** [comment]
Need reproducion

**Reply by cgx452306705** [reply]
Need reproducion

没明白....需要复制什么.... 
![image](https://user-images.githubusercontent.com/12523996/220276063-207069e4-0fac-4c50-936e-4bed4cc061ac.png)

**Reply by Mister-Hope** [reply]
https://minimum-reproduction.wtf/

**Reply by meteorlxy** [reply]
<img width="783" alt="image" src="https://user-images.githubusercontent.com/18205362/220281352-fd6af5e0-7159-4bd4-99d3-04aadfa657fe.png">

**Comment by meteorlxy** [comment, marked as answer]
Please read the changelog. There should be lots of breaking changes from beta.50 to beta.60
